executor/distsql.go

```diff
@@ -348,6 +353,15 @@ type IndexLookUpExecutor struct {
 	colLens         []int
 }
 
+type checkIndexValue struct {
+	isCheckOp bool
+	tps       []*types.FieldType
+	tbl       table.Table
```

Comment:
These fields are duplicated with IndexLookUpExecutor's fields, why not just use the fields in IndexLookUpExecutor?